explore-analyze/alerts-cases/cases.md

```diff
@@ -26,3 +26,13 @@ If you create cases in the {{observability}} or {{security-app}}, they are not v
 * [Configure access to cases](cases/setup-cases.md)
 * [Open and manage cases](cases/manage-cases.md)
 * [Configure case settings](cases/manage-cases-settings.md)
+
```

Comment:
I can't comment on the exact line, but in line 15, there's a "[preview]" tag that doesn't do anything in these docs. Not sure if that feature is in tech preview and we want to add an `applies_to` tag.

Reply:
Yeah, I was thinking of replacing that with the applies to tag. Just need to check with the Ux Management folks first.